software_reseller/data/ir_actions_act_window.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo>
+    <data noupdate="1">
```

Comment:
actions should probably be updatable if then need to be fixed after migration.

Example:
- They migrates
- Actions are disabled because based on fields that does not exist (wrong domain)
- They just reinstall the module in their new version and get fixed